ui/raidboss/data/07-dt/trial/necron-ex.txt

```diff
@@ -0,0 +1,260 @@
+### THE MINSTREL'S BALLAD: NECRON'S EMBRACE
+# ZoneId: 1296
+
+# -ii AE09 AE0A AE11 AE12 AE13 AE1E AE30 AE32 AE34 AE35 AE39 AE3C AE3F AE47 AF13
+
+hideall "--Reset--"
+hideall "--sync--"
+
+# Base timeline pulled from FFLogs enrage pull
+# TODO: Figure out phase syncs / push timings?
+# TODO: Duration on Memento Mori middle line?
+# TODO: Adds phase tankbuster timing varies, don't have all possible times
+# Maybe it doesn't make sense to include these in timeline?
+
+0.0 "--Reset--" ActorControl { command: "4000000F" } window 0,100000 jump 0
+
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+8.0 "--sync" StartsUsing { id: "AE30", source: "Necron" }
+14.8 "Blue Shockwave" Ability { id: "AE31", source: "Necron" }
+18.9 "Blue Shockwave" Ability { id: "AE31", source: "Necron" }
+
+# Fear of Death 1
+28.9 "Fear of Death (cast)" Ability { id: "AE06", source: "Necron" }
+31.9 "Fear of Death (damage)" Ability { id: "AE07", source: "Necron" }
+34.7 "Choking Grasp (puddles)" Ability { id: "AE08", source: "Icy Hands" }
+40.0 "Cold Grip" Ability { id: "AE44", source: "Necron" }
+41.7 "Existential Dread" Ability { id: "AE0B", source: "Necron" }
+
+# Memento Mori 1
+52.4 "Memento Mori" Ability { id: ["AE15", "AE16"], source: "Necron" }
+58.6 "Choking Grasp" Ability { id: "AE17", source: "Icy Hands" }
+59.5 "Smite of Gloom" Ability { id: "AE3A", source: "Necron" }
+
+69.6 "Soul Reaping" Ability { id: "AE0C", source: "Necron" }
+77.7 "Twofold Blight/Fourfold Blight" Ability { id: ["AE0D", "AE0E"], source: "Necron" }
+78.9 "Aetherblight" Ability { id: ["AE40", "B07F", "B080", "B081"], source: "Necron" }
+79.0 "Shockwave" Ability { id: ["AE0F", "AE10"], source: "Necron" }
+
+# Fear of Death 2
+84.7 "Soul Reaping" Ability { id: "AE0C", source: "Necron" }
+92.8 "Fear of Death (cast)" Ability { id: "AE06", source: "Necron" }
+95.8 "Fear of Death (damage)" Ability { id: "AE07", source: "Necron" }
+98.7 "Choking Grasp (puddles)" Ability { id: "AE08", source: "Icy Hands" }
+102.9 "The End's Embrace" #Ability { id: "AE36", source: "Necron" }
+109.0 "Choking Grasp" #Ability { id: "AE17", source: "Icy Hands" }
+114.0 "Blue Shockwave" Ability { id: "AE31", source: "Necron" }
+118.1 "Blue Shockwave" Ability { id: "AE31", source: "Necron" }
+124.1 "Twofold Blight/Fourfold Blight" Ability { id: ["AE0D", "AE0E"], source: "Necron" }
```

Comment:
Here and elsewhere, it might be worth adding an English `replaceText` of `Twofold/Fourfold Blight`.